### PR TITLE
Reaction Content Type

### PR DESCRIPTION
**Pull request by nplasterer**
Closes https://github.com/xmtp/xmtp-android/issues/93

This adds the reaction content type to kotlin allowing decryption of reaction messages in push notifications.

### CODE REVIEW

File: library/src/main/java/org/xmtp/android/library/codecs/GroupChatMemberAddedCodec.kt [hunk 1]
```diff
@@ -1,11 +1,7 @@
-package uniffi.xmtp_dh.org.xmtp.android.library.codecs
```
Comment (nplasterer) [inline library/src/main/java/org/xmtp/android/library/codecs/GroupChatMemberAddedCodec.kt:1]: These some how got added under the Rust package. Most of these package changes are just a clean up item.

File: library/src/main/java/org/xmtp/android/library/codecs/ReactionCodec.kt [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    val schema: ReactionSchema,
+)
+
+enum class ReactionAction {
+    ADDED, REMOVED
+}
+
+enum class ReactionSchema {
+    UNICODE, SHORTCODE, CUSTOM
+}
```
Comment (rygine) [inline library/src/main/java/org/xmtp/android/library/codecs/ReactionCodec.kt:26]: a brief googling on kotlin enum classes says that these will yield UPPERCASE strings. the TypeScript types for `action` and `schema` expect lowercase values. sorry if my understanding of this is wrong.

Comment (nplasterer) [reply library/src/main/java/org/xmtp/android/library/codecs/ReactionCodec.kt:26]: Yep totally. I just changed them to lowercase.

**Review by daria-github — APPROVED** [review]
LGTM!

**Pull request merged**